### ISSUE TITLE
``affected_version_range`` should be optional

### ISSUE THREAD

**Issue by Hritik14** [nexB/vulnerablecode#628]
Not all data sources provide an affected version range, there could only be a fixed version. This should be made optional with ``None`` as default
https://github.com/nexB/vulnerablecode/blob/283ef0d8878b953b88fb12043b962ab6c59bbb22/vulnerabilities/importer.py#L116-L125

Subsequently, the following needs to be fixed for ``NoneType``
https://github.com/nexB/vulnerablecode/blob/283ef0d8878b953b88fb12043b962ab6c59bbb22/vulnerabilities/importer.py#L130-L136

```
Reported-By: Tushar Goel <tushar.goel.dav@gmail.com>
```

**Comment by purna135** [comment]
I attempted to resolve this issue. 
First I followed the environment installation instructions without using Docker Compose, but after installing all dependencies and running pytest, I am seeing the following errors:
```
vulnerabilities/tests/test_basics.py:48: Exception
-------------------------------------------- Captured stderr call --------------------------------------------
All done! ✨ 🍰 ✨
103 files would be left unchanged.
ERROR: /home/Rama/Downloads/OpenSource/vulnerablecode/vulnerabilities/api.py Imports are incorrectly sorted and/or formatted.
============================================== warnings summary ==============================================
venv/lib/python3.9/site-packages/django/conf/__init__.py:222
  /home/Rama/Downloads/OpenSource/vulnerablecode/venv/lib/python3.9/site-packages/django/conf/__init__.py:222: RemovedInDjango50Warning: The USE_L10N setting is deprecated. Starting with Django 5.0, localized formatting of data will always be enabled. For example Django will display numbers and dates using the format of the current locale.
    warnings.warn(USE_L10N_DEPRECATED_MSG, RemovedInDjango50Warning)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
========================================== short test summary info ===========================================
FAILED vulnerabilities/tests/test_basics.py::BaseTests::test_codestyle - Exception: Unsorted imports, pleas...
================================== 1 failed, 20 passed, 1 warning in 10.46s ==================================
```
Could you possibly assist me in determining where I am making a mistake?

**Comment by Hritik14** [comment]
Hello @purna135
Thanks for giving it a try, although this has been addressed in https://github.com/nexB/vulnerablecode/pull/623 already.

Also, we've recently did a huge upgrade to our docs, you read the full docs (some parts still in development) at https://vulnerablecode.readthedocs.io/en/latest/ :)

**Comment by purna135** [comment]
Thanks @Hritik14, since this has been addressed in #623 already, do I need to do anything in this issue?

**Comment by Hritik14** [comment]
Nope
